SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -335,4 +335,11 @@ var splunkRumInitializeCalledTime = Date()
         integrateWebViewWithBrowserRum(view: view)
     }
 
+    /**
+       This check is to determine whether the splunkrum library has been initialized
+     */
+    public class func rumInitialize() -> Bool? {
```

Comment:
Perhaps rename it to `isInitialized`, currently it looks as if it does the initialization

Reply:
Done